server/src/api/community.rs

```diff
@@ -264,6 +264,17 @@ impl Perform for Oper<CreateCommunity> {
       return Err(APIError::err("site_ban").into());
     }
 
+    // Double check for duplicate community actor_ids
+    let actor_id = make_apub_endpoint(EndpointType::Community, &data.name).to_string();
+    let actor_id_cloned = actor_id.to_owned();
```

Comment:
Is this really needed?

Reply:
Ya unfortunately, bc one of them gets moved into the `blocking` and can't get extracted out again.